keep-ui/app/alerts/alert-sidebar.tsx

```diff
@@ -3,10 +3,10 @@ import { Dialog, Transition } from "@headlessui/react";
 import { AlertDto } from "./models";
 import { Button, Title, Card, Badge } from "@tremor/react";
 import { IoMdClose } from "react-icons/io";
-import AlertMenu from "./alert-menu";
-import GraphVisualization from "./alert-graph-viz";
+// import AlertMenu from "./alert-menu";
```

Comment:
forgotten?

Reply:
Not really, it's a "todo" left from `alert-sidebar` feature and import was redundant here since it's not being used (it's kinda automatic with linter)